### DISCUSSION TITLE
Data Grid code sandbox is broken

### DISCUSSION THREAD

**Discussion by dsnair** [twilio-labs/paste#3809] [Q&A] [answered]
Hello,

I am trying to develop some features using the Data Grid. However, its [code sandbox](https://paste.twilio.design/components/data-grid#data-grid-with-selectable-rows) is broken with the error `Could not find module in path: '@twilio-paste/core/theme' relative to '/src/index.tsx'`. Can you please take a look?

Thanks!

**Comment by roxanagomez** [comment]
Hey @dsnair we will look into this and get back to you.

**Comment by TheSisb** [comment, marked as answer]
Hi @dsnair 

That's a common error on Codesandbox unfortunately. Just refresh and it should go away.  It works everywhere, just sometimes flakes on the Codesandbox platform.
